classes/Routeur.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Class Routeur
+ * 
+ * Créé les routes et trouve les controlleurs
+ */
+
+include_once("controllers/Frontend.php");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ["include_once" statement detected. File manipulations are discouraged. Statement is not a function, no parentheses are required.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)